uikit/views/src/main/java/com/sygic/maps/uikit/views/common/extensions/ContextExtensions.kt

```diff
@@ -122,16 +131,8 @@ fun Context.copyToClipboard(text: String) {
     }
 }
 
-fun Context.shortToast(@StringRes text: Int) {
-    Toast.makeText(this, text, Toast.LENGTH_SHORT).show()
-}
+fun Context.shortToast(@StringRes text: Int) = Toast.makeText(this, text, Toast.LENGTH_SHORT).show()
+fun Context.longToast(@StringRes text: Int) = Toast.makeText(this, text, Toast.LENGTH_LONG).show()
 
-fun Context.longToast(@StringRes text: Int) {
-    Toast.makeText(this, text, Toast.LENGTH_LONG).show()
-}
-
-inline val Context.locationManager: LocationManager?
-    get() = getSystemService(LOCATION_SERVICE) as? LocationManager
-
-inline val Context.clipboardManager: ClipboardManager?
-    get() = getSystemService(CLIPBOARD_SERVICE) as? ClipboardManager
\ No newline at end of file
+fun Context.showKeyboard() = inputMethodManager?.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT, 0)
```

Comment:
`toggle`? To 2x po sebe predsa aj schova, to skor `showSoftInput` ak to ma robit co nazov hovori ;)